docs/src/prototypes/hexagonalAvatar/CustomAvatar.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Avatar as StardustAvatar, AvatarProps, callable } from '@stardust-ui/react'
+
+const Avatar = (props: AvatarProps & { hexagonal?: boolean }) => {
+  const { hexagonal, variables: propVariables, ...rest } = props
+  const variables = siteVars => {
+    return {
+      ...callable(propVariables)(siteVars),
+      ...(hexagonal && {
+        // these values are defined for the medium size avatar,
+        // but can be exptended if we need to support different
+        // hexagonal avatar sizes
+        width: '36px',
+        height: '32px',
+        clipPath: siteVars.hexClipPath,
+        borderRadius: '0px',
+        avatarBorderColor: '',
+        avatarBorderWidth: 0,
+      }),
+    }
+  }
+  return <StardustAvatar {...rest} variables={variables} />
```

Comment:
You should use `mergeComponentVariables` here instead of the callable above.

Reply:
But we are not exporting mergeComponentVariables currently, that's why I needed to do this... Or was this changed recently?